src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/DeclarationTests.cs

```diff
@@ -133,7 +133,7 @@ .maxstack  4
             });
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/13159")]
+        [Fact]
         [WorkItem(13159, "https://github.com/dotnet/roslyn/issues/13159")]
         public void ExpressionLocals_ExpressionStatement_02()
```

Comment:
Please add tests similar to ExpressionLocals_Assignment_01 and ExpressionLocals_LocalDeclarationStatement_01.